docs/src/jekyll/docs.md

```diff
@@ -0,0 +1,546 @@
+---
+layout: docs
+---
+
+# Introduction
+
+Oftentimes, our applications read and manipulate data from a variety of
+different sources such as databases, web services or file systems. These data
+sources are subject to latency, and we'd prefer to query them efficiently.
+
+If we are just reading data, we can make a series of optimizations such as:
+
+ - batching requests to the same data source
+ - requesting independent data from different sources in parallel
+ - caching previously seen results
+
+However, if we mix these optimizations with the code that fetches the data
+we may end up trading clarity for performance. Furthermore, we are
+mixing low-level (optimization) and high-level (business logic with the data
+we read) concerns.
+
+Fetch is a library that allows your data fetches to be written in a concise,
+composable way while executing efficiently. You don't need to use any explicit
+concurrency construct but existing idioms: applicative for concurrency and
+monad for sequencing.
+
+# Installation
+
+First of all, add the following dependency to your SBT build file:
+
+```scala
+"org.fortysevendeg" %%% "fetch" % "0.1.0"
+```
+
+Now you'll have Fetch available both in Scala and Scala.js.
```

Comment:
The dependency declaration with `%%%` should cover both Scala and Scala.js but i'll include all possibilities: scala only, scala.js only and both.